rskj-core/src/main/java/co/rsk/peg/storage/FederationStorageIndexKey.java

```diff
@@ -12,18 +12,21 @@ public enum FederationStorageIndexKey {
     NEW_FEDERATION_KEY("newFederation"),
     OLD_FEDERATION_KEY("oldFederation"),
     PENDING_FEDERATION_KEY("pendingFederation"),
+    PROPOSED_FEDERATION_KEY("proposedFederation"),
```

Comment:
We should probably get rid of the _KEY suffix. Getting rid of it in all the keys might have a big impact so we could do it in a separate task. But let's at least not add it to new keys

Reply:
yes, i've signed up a note for doing exactly that. But i'll not add this `_KEY`, you right.

Reply:
Done. And i created the task to take care of the rest